test/aws/default/build/iam.tf

```diff
@@ -99,4 +99,51 @@ resource "aws_iam_group" "administrators" {
 
 output "iam_group_administrators" {
   value = "${aws_iam_group.administrators.name}"
+}
+
+#======================================================#
+#                    IAM Roles
+#======================================================#
+
+resource "aws_iam_role" "role_for_config_delivery_channel" {
```

Comment:
I think I'd prefer to keep the supporting fixtures for Config in config.tf .  In other words, move the iam_role and iam_role_profile into config.tf.  The idea being, the Terraform files group together all of the pieces needs to test closely related InSpec resources.  So, it's not "what it is", but rather, "what it is used for"